docs/learn/learn-elastic-scaling.md

```diff
@@ -75,11 +67,52 @@ The diagram above shows how four parablocks are backed and included in the relay
+
+- **Flexible scaling for new applications**: Elastic scaling allows applications to start small with On-Demand Coretime and scale up seamlessly beyond one core when demand grows. This prevents over-allocation (wasting money on unused resources) or under-allocation (causing performance issues), both common in traditional setups.
+
+- **Resource optimization**: Parachains can load-balance their core usage by leasing additional cores when demand increases and releasing them when activity subsides. Secondary markets for coretime also ensure that unused resources are reallocated efficiently, optimizing overall system performance.
+
+- **Improved user experience**: End-users enjoy smooth and reliable performance, even during peak traffic. Applications no longer face the risk of slowing down or freezing when demand spikes, ensuring a consistent, high-quality experience.
+
+Elastic scaling increases the single-shard performance. Polkadot’s approach to scaling has traditionally been to scale via multiple parachains, called “horizontal scaling” or “horizontal sharding”. However, setting up and parallelizing processes over multiple parachains and opening channels via XCM takes time and effort. With elastic scaling, projects can benefit from higher throughput before parallelizing processes over multiple parachains.
+
+## Elastic Scaling Use Cases
+
+- **"Bursty” applications**: Many applications experience sporadic spikes in activity rather than constant demand, e.g., Gaming, DeFi, Auctions, Messaging, Social Media, etc. With elastic scaling, these applications can acquire additional coretime during peak usage and release it during off-peak hours, optimizing performance while reducing costs.
+
+- **Growing applications & startups**: Startups and new projects often face unpredictable user growth. Elastic scaling allows them to start with minimal resources (e.g., a single core) and scale seamlessly as user demand increases before scaling horizontally, thus preventing overcommitting resources upfront. This ensures applications can grow efficiently while maintaining performance and cost-effectiveness.
+
+- **Large-scale IoT applications**: IoT systems require massive scalability to process data from millions of devices. Elastic scaling enables parachains to process high transaction volumes cost-effectively, while Polkadot’s model ensures the privacy and security needed in IoT environments.
+
+- **Real-time applications**: Some applications require low latency and real-time performance (e.g., gaming, payments / financial services, or data feeds). These benefit from elastic scaling’s ability to reduce latency down to 2s. This ensures smooth and reliable performance during high-demand scenarios.
+
+## Elastic Scaling Roadmap
```

Comment:
Maybe mention that it is already on Kusama (in MVP format?)

Reply:
I would refrain from adding things that have a short lifespan and require us to update frequently. Stuff like "currently live on Kusama" or "this new feature", etc. Imo we should aim to get the wiki as static as possible.